### ISSUE TITLE
Text spacing and viewport requirements

### ISSUE THREAD

**Issue by kimviens** [w3c/wcag#3409]
Hello everyone,
I would like to know if it is possible to be given a recommended viewport size when testing 1.4.12 Text Spacing. The reason why I ask is because when I do WCAG testing, I normally use a width of 1280px like it is recommended by 1.4.10 Reflow. But when I use my normal viewport size (1496px of width), I get different results.

This means that a website might fail 1.4.12 Text spacing when using 1280px causing text to be clipped, but when using my normal viewport size that text is no longer clipped.

Is the website supposed to pass this criteria no matter the viewport size? Or is the viewport size of 1280px is implied as the standard to test?

**Comment by patrickhlauke** [comment]
> Is the website supposed to pass this criteria no matter the viewport size?

Yes

**Comment by kimviens** [comment]
Excellent, that is all I needed. Thank you @patrickhlauke!
